### ISSUE TITLE
Grouping error: 7 ERROR: column "giftvoucher_vouchers.postDate"

### ISSUE THREAD

**Issue by Tam** [verbb/gift-voucher#6]
### Description
When trying to save a commerce product with a voucher field, I get the following error:

```
SQLSTATE[42803]: Grouping error: 7 ERROR: column "giftvoucher_vouchers.postDate" must appear in the GROUP BY clause or be used in an aggregate function
LINE 13: ORDER BY "postDate" DESC
^
The SQL being executed was: SELECT COUNT(*)
FROM (SELECT "elements"."id" AS "elementsId", "elements_sites"."id" AS "elementsSitesId", "content"."id" AS "contentId"
FROM "elements" "elements"
INNER JOIN "giftvoucher_vouchers" "giftvoucher_vouchers" ON "giftvoucher_vouchers"."id" = "elements"."id"
INNER JOIN "elements_sites" "elements_sites" ON "elements_sites"."elementId" = "elements"."id"
INNER JOIN "content" "content" ON "content"."elementId" = "elements"."id"
WHERE ("elements_sites"."siteId"=1) AND ("content"."siteId"=1) AND ("elements"."id"='604') AND ("elements"."archived"=FALSE)
ORDER BY "postDate" DESC) "subquery"
INNER JOIN "giftvoucher_vouchers" "giftvoucher_vouchers" ON "giftvoucher_vouchers"."id" = "subquery"."elementsId"
INNER JOIN "elements" "elements" ON "elements"."id" = "subquery"."elementsId"
INNER JOIN "elements_sites" "elements_sites" ON "elements_sites"."id" = "subquery"."elementsSitesId"
INNER JOIN "content" "content" ON "content"."id" = "subquery"."contentId"
ORDER BY "postDate" DESC
```

### Steps to reproduce

1. Create a commerce product and a voucher.
2. Add the voucher to the product via a Voucher field, and save.

### Additional info

Name | Version
-- | --
PHP version | 7.1.19
PostgreSQL | 9.5.14
Craft Pro | 3.0.29
Craft Commerce | 2.0.0-beta.13.1
Gift Voucher | 2.0.2

**Comment by Tam** [comment]
The issue goes away if I comment out:
https://github.com/verbb/gift-voucher/blob/craft-3/src/elements/db/VoucherQuery.php#L167-L169

**Comment by engram-design** [comment]
Fixed in [2.0.3](https://github.com/verbb/gift-voucher/releases/tag/2.0.3)
